samples/Playground/Playground.Shared/App.xaml.cs

```diff
@@ -185,6 +188,10 @@ protected async override void OnLaunched(LaunchActivatedEventArgs args)
 		if (logger.IsEnabled(LogLevel.Warning)) logger.LogWarningMessage("LogLevel:Warning");
 		if (logger.IsEnabled(LogLevel.Error)) logger.LogErrorMessage("LogLevel:Error");
 		if (logger.IsEnabled(LogLevel.Critical)) logger.LogCriticalMessage("LogLevel:Critical");
+
+		ts = new ThemeService(_window, new Dispatcher(_window));
```

Comment:
ThemeService should be registered in the depedency container (see ConfigureServices in app.xaml.host.cs) and then retrieved using Host.Services.GetService(IThemeService)

Reply:
done